content/en/docs/releases/supported-releases/index.md

```diff
@@ -30,10 +30,16 @@ assistance. Separately, 3rd parties and partners may offer longer-term support s
 |Type              | Support Level                                                                                                         | Quality and Recommended Use
 |------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------
 |Development Build | No support                                                                                                            | Dangerous, may not be fully reliable. Useful to experiment with.
-|Minor Release     | Support is provided until 3 months after the next minor release                                                       | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
+|Minor Release     | Support is provided until 4 and a half months after the next minor release                                            | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
 |Patch             | Same as the corresponding Minor release                                                                               | Users are encouraged to adopt patch releases as soon as they are available for a given release.
 |Security Patch    | Same as a Patch, however, it will not contain any additional code other than the security fix from the previous patch | Given the nature of security fixes, users are **strongly** encouraged to adopt security patches after release.
 
+{{< tip >}}
+Minor releases occur every 3 months and are supported for 4 and a half months after the next
```

Comment:
Tweak this as well based on prior comment. I would just remove the part about 4.5 months altogether.